weekThree/file_manager.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+class FileManager:
+    def __init__(self, file_path):
+        self.file_path = file_path
+
+    def count_lines(self):
+        if not os.path.exists(self.file_path):
+            print("File not found!")
+            return 0
+        with open(self.file_path, 'r', encoding='utf-8') as file:
+            return len(file.readlines())
+
+    def write_data(self, data):
+        with open(self.file_path, 'w', encoding='utf-8') as file:
```

Comment:
You can improve write handling in file operations (e.g., check that the file exists before accessing it, so you do not overwrite some "important" file).